assembler/src/spades_pipeline/support.py

```diff
@@ -20,7 +20,10 @@
 import tempfile
 import traceback
 from platform import uname
-from distutils.version import LooseVersion
+try:
+    from packaging.version import Version
+except ModuleNotFoundError:
```

Comment:
(we support 2.7, 3.6-3.10)
Supported only since 3.6 

Better use: ImportError